### DISCUSSION TITLE
Couldn't clone reposetory in github

### DISCUSSION THREAD

**Discussion by kritakaryal** [community/community#86981] [Repositories] [answered]
### Select Topic Area

Question

### Body

I am having problems regarding the cloning of reposetory in github feature for creating new reposetory.
It is constantly saying the reposetory is big or couldnt clone
![Uploading 4D5E4C78-1419-452C-93AE-4ED38E2EBD89.png…]()

**Comment by NS-Sheam** [comment]
<img width="331" alt="gt1" src="https://github.com/community/community/assets/117335351/fdd0d6f4-1590-4b13-8f16-3a8b1122e636">
 
See the image. Go to your terminal or powershell and write-
git clone https://repo-url.git

**Comment by faridnec** [comment, marked as answer]
If the repository is too large, consider using the Git protocol or SSH to clone the repository. Instead of using HTTPS, try cloning the repository using the Git protocol or SSH.
```bash
git clone git@github.com:username/repository.git
```
or partial clone to fetch only the parts you need. Git supports a shallow clone to limit the commit history or depth, which can reduce the size of the repository.
```bash
git clone --depth 1 https://github.com/username/repository.git
```

**Reply by kritakaryal** [reply]
Ok i will try that as well, thanks. 
and how long of repo can i clone like that?

**Reply by faridnec** [reply]
it depends on the size of the repository and the available resources on your local machine and network.
